.ddev/config.yaml

```diff
@@ -1,33 +1,31 @@
 name: tea
 type: php
 docroot: ""
-php_version: "8.1"
+php_version: "8.3"
```

Comment:
Please let's move the PHP upgrade to a separate PR. (Also, we need to stay below PHP 8.3 in order to not break PHP-CS-Fixer at this moment.)

Reply:
Ok